### PR TITLE
 Backend MVP: Initial Release

### PR DESCRIPTION
**Pull request by amirabayoumi**
Core PHP Functionality

### CODE REVIEW

File: Dump20241224.sql [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  `pass` int(11) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=2 DEFAULT CHARSET=utf8;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Table structure for table `catogery`
+--
+
+DROP TABLE IF EXISTS `catogery`;
```
Comment (AnouckGaloppin) [inline Dump20241224.sql:37]: typo! : it should be category

File: Dump20241224.sql [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `catogery` (
```
Comment (AnouckGaloppin) [inline Dump20241224.sql:40]: Same as before

File: Dump20241224.sql (hunk 1 above)
Comment (amirabayoumi) [reply Dump20241224.sql:37]: @AnouckGaloppin thanks for your review will fix it

Comment (AnouckGaloppin) [reply Dump20241224.sql:37]: I also found it in functions, so maybe use CTRL+SHIFT+F to make sure you don't miss any

Comment (amirabayoumi) [reply Dump20241224.sql:37]: Yes , I normally copy past exact tables names. Thanks for your advice ✔️

**Review by ViktoriiaIsk — APPROVED** [review]

**Pull request merged**